css/css-writing-modes/font-relative-units-001.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>writing-modes and font units on table rows / upright</title>
```

Comment:
“upright vertical writing-mode and ch unit on table rows” would be clearer and more precise, I think.

Also s/font-relative-units-001/ch-units-vrl-001/ - shorter and yet more specific! I'd also accept ch-units-001, if you want to be a bit vaguer for some reason, but there are a lot of font-relative units and it's good to be specific which one we are testing, since we are not testing all of them.